api/src/main/java/org/openmrs/PatientIdentifier.java

```diff
@@ -328,4 +331,21 @@ public int compare(PatientIdentifier pi1, PatientIdentifier pi2) {
 			return retValue;
 		}
 	}
+
+
+	/**
+	 * @since 2.3.6
+	 * @return patientProgram
+	 */
+	public PatientProgram getPatientProgram() {
+		return patientProgram;
+	}
+
+	/**
+	 * @since 2.3.6
```

Comment:
It would be great to add a description to this method.

Reply:
@dkayiwa done